kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -141,58 +48,23 @@
               :primary="false"
               @click="toggleFolderSearchSidePanel('search')"
             />
-
           </div>
+
           <!-- default/preview display of nested folder structure, not search -->
           <div v-if="!displayingSearchResults" data-test="topics">
-            <!-- display for each nested topic/folder  -->
-            <div v-for="t in topicsForDisplay" :key="t.id">
-              <!-- header link to folder -->
-              <h2>
-                <KRouterLink
-                  :text="t.title"
-                  :to="genContentLink(t.id)"
-                  class="folder-header-link"
-                  :appearanceOverrides="{ color: $themeTokens.text }"
-                >
-                  <template #iconAfter>
-                    <KIcon
-                      icon="chevronRight"
-                      :style="{ top: '4px' }"
-                    />
-                  </template>
-                </KRouterLink>
-              </h2>
-              <!-- card grid of items in folder -->
-              <LibraryAndChannelBrowserMainContent
-                v-if="t.children && t.children.length"
-                data-test="children-cards-grid"
-                :contents="t.children"
-                currentCardViewStyle="card"
-                :gridType="2"
+            <!-- Rows of cards and links / show more for each Topic -->
+            <template v-for="t in topicsForDisplay">
+              <TopicSubsection
+                :key="t.id"
+                :topic="t"
+                :subTopicLoading="subTopicLoading"
```

Comment:
Maybe we could change `subTopicLoading` prop on `TopicSubsection` to be a boolean, and then do the equality check here?
```suggestion
                :subTopicLoading="t.id === subTopicLoading"
```

That way the TopicSubsection component doesn't need to worry about things that are outside its purview.